### ISSUE TITLE
What is the learning strategy when you finetune model for the metric depth estimation?

### ISSUE THREAD

**Issue by lmomoy** [LiheYoung/Depth-Anything#57]

**Comment by LiheYoung** [comment]
As we mentioned [here](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth) and in our paper, we follow ZoeDepth to fine-tune our model for metric depth estimation.
